Dockerfile

```diff
@@ -40,7 +40,7 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7
 
 # shadow-utils is required for `groupadd`, etc.
 RUN PKGS="libgcc shadow-utils" \
-    && microdnf install --nodocs $PKGS \
+    && microdnf -y install --nodocs $PKGS \
```

Comment:
I prefer the verbose form for flags in scripts. 

```suggestion
    && microdnf --assumeyes install --nodocs $PKGS \
```

Reply:
<img width="1024" alt="Screenshot 2023-05-10 at 12 04 57" src="https://github.com/Kuadrant/limitador/assets/1842261/5cc1e22d-1937-455a-aecf-a50513e492e7">

BUSTED! 😂

Reply:
tsk tsk tsk